alm/Dockerfile

```diff
@@ -19,5 +19,11 @@ ARG DOT_ENV_PATH=./alm/.env
 COPY ${DOT_ENV_PATH} ./alm/.env
 
 WORKDIR /mono/alm
-CMD echo "To start the application run:" \
-  "yarn start"
+RUN yarn run build
+
+
+FROM node:12 as alm-production
+RUN yarn global add serve
+WORKDIR /app
+COPY --from=alm-builder /mono/alm/build .
+CMD echo "To start the application run: serve -p 8080 -s ."
```

Comment:
could you check that this form of running the application will allow to use it with the Heroku?

Reply:
Updated the CMD command and the variable used, to make it work with Heroku https://github.com/poanetwork/tokenbridge/pull/380/commits/493e16c4f8e0db6afccb7efabbbeed80947513f3